lib/ansible/plugins/test/changed.yml

```diff
@@ -14,7 +14,7 @@ DOCUMENTATION:
       required: True
 EXAMPLES: |
   # test 'status' to know how to respond
-  {{ (taskresults is changed }}
+  {{ taskresults is changed }}
```

Comment:
There are other occurrences with the same error. While you are here can you fix them as well? Thanks,

Reply:
@Akasurde done